.github/workflows/test-windows.yml

```diff
@@ -0,0 +1,97 @@
+name: test-windows
+on:
+  pull_request:
+    paths-ignore:
```

Comment:
here I think we can use [paths:](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths) instead of paths-ignore, since the gotestsum command targets these specific packages:

```
nomad/drivers/docker
nomad/client/lib/fifo
nomad/client/logmon
```

(note there is a "nomad" dir in this nomad repo, so do need the leading "nomad/")

Reply:
I'm second-guessing myself now, because those packages have dependencies on other things (also would not need leading "nomad/" because those packages are not in the nomad dir...)

So I think instead we stick with the same list as test-core, though I wish we didn't have to maintain the same list separately...